harc_game_web/apps/wotd/views.py

```diff
@@ -0,0 +1,40 @@
+from django.shortcuts import render
+from django.views import View
+from django.utils import timezone
+from django.db.models import F
+
+import datetime
+
+from apps.wotd.models import WordOfTheDay
+
+class WordOfTheDayView(View):
+
+    def _verify(self, word, guess):
+        return (word.answer == guess)
+
+    def get(self, request, *args, **kwargs):
+        """
+        Hasła dnia
+            Dzisiejsze hasło dnia
+            Jutrzejsze hasło dnia
+            Formularz do sprawdzania poprawności hasła dnia
+            Poprzednie hasła dnia z odpowiedziami
+
+        """
+
+        # Firltrowanie pro datach
+        words_of_the_past = WordOfTheDay.objects.all().exclude(date__gte = timezone.now()).order_by(F('date').desc(nulls_last=True))
+        word_of_the_day = WordOfTheDay.objects.filter(date = timezone.now())[0]
+        word_for_tomorrow = WordOfTheDay.objects.filter(date = timezone.now() + datetime.timedelta(days=1))[0]
```

Comment:
in line 27 and 28 you can use what you already filtered out (`words_of_the_past `) to filter further, i.e.:
```
word_of_the_day  = words_of_the_past.filter(date = timezone.now())[0]
```